docs/docusaurus.config.js

```diff
@@ -0,0 +1,133 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Libretime',
+  tagline: 'Broadcast without limits',
+  url: 'https://your-docusaurus-test-site.com',
+  baseUrl: '/libretime/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+  organizationName: 'zklosko', // Usually your GitHub org/user name.
+  projectName: 'Libretime', // Usually your repo name.
+  trailingSlash: true,
+
+  plugins: [
+    [
+      require.resolve("@cmfcmf/docusaurus-search-local"), {
+        // Options
+        indexBlog: false,
+        indexPages: false,
+      }
+    ]
+  ],
+
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en'],
```

Comment:
I saw some English files in the i18n/fr/** dir. Those should probably be moved.

Reply:
Those files have been removed.